test/test-color-scheme.rb

```diff
@@ -78,5 +80,35 @@ def test_default
       assert_equal(expected_schema_keys.sort,
                    Test::Unit::ColorScheme.default.to_hash.keys.sort)
     end
+
+    def test_available_colors
+      terms = {
+        "rxvt"                 => 8,
+        "xterm-color"          => 8,
+        "alacritty"            => 256,
+        "iTerm.app"            => 256,
+        "screen-256color"      => 256,
+        "screenxterm-256color" => 256,
+        "tmux-256color"        => 256,
+        "vte-256color"         => 256,
+        "vscode-direct"        => 2**24,
+        "vte-direct"           => 2**24,
+        "xterm-direct"         => 2**24,
+      }
+      terms.each do |term, expected_available|
+        ENV["TERM"] = term
+        assert_equal(expected_available,
+                     Test::Unit::ColorScheme.available_colors,
+                     "Incorrect available_colors for TERM=%s" % [term])
+      end
+    end
+
+    def test_default_for_direct_color
+      ENV["TERM"] = "xterm-direct"
+      assert_equal(2**24, Test::Unit::ColorScheme.available_colors)
```

Comment:
This is needless.
This is tested in other test.

Reply:
Good point. Deleted.